Packages/Tracking/Physical Hands/Runtime/Scripts/PhysicalHandsButton.cs

```diff
@@ -22,7 +22,7 @@ public class PhysicalHandsButton : MonoBehaviour
 
```

Comment:
We should either mark this script as obsolete, delete it, or re-use it for the new button script

(as we are going to do a breaking change for this release anyway)

Reply:
I think we just delete if going to be breaking, then we can re-name the regular button. with all of the auto stuff, any buttons using the old script should work pretty much off the bat anyway

Reply:
Makes sense to me